backend/dataall/cdkproxy/stacks/policies/_lambda.py

```diff
@@ -1,9 +1,13 @@
+from dataall.db import permissions
```

Comment:
[Question] dataall.db permissions is still a file that remains common to all modules right? @nikpodsh How would this piece work for users? Do they have to all modify this file and add their permissions?

Reply:
I think this module will migrate to core and will have only some basic permissions that needed for the core part to function well. Then every module will register their permissions into the core part (by adding them into permission containers). I can define a better API for the permissions if you think that would be helpful